arch/xtensa/src/lx6/Toolchain.defs

```diff
@@ -55,8 +59,15 @@ else
   MAXOPTIMIZATION += -fomit-frame-pointer
 endif
 
-ifeq ($(CONFIG_MM_KASAN),y)
-  ARCHCPUFLAGS += -fsanitize=kernel-address
```

Comment:
why change these lines

Reply:
Just to move the usage of `ARCHCPUFLAGS` closer to its declaration.